virttest/backends/virttools/cfg/tests.cfg

```diff
@@ -0,0 +1,24 @@
+# File reserved for avocado-vt use, don't modify.
+include tests-shared.cfg
+
+variants:
+    - @libvirt_quick:
+        start_vm = no
+        virt_install_binary = /usr/bin/virt-install
+        qemu_img_binary = /usr/bin/qemu-img
+        hvm_or_pv = hvm
+        use_os_variant = yes
+        use_os_type = yes
+        only qcow2
```

Comment:
Do we really need below filters when you enable virttools type?

Reply:
@chunfuwen In short, yes, I believe they are necessary, at least I prefer to keep them. I tried to run tests without them and I conclude it's better to keep them.

Details:
-----------
I believe they are not necessary _functionally_. However, I've found they have a big impact regarding performance. When I tried test case without these filters, they never started in a reasonable time while I found the cartesian config was still creating its test dictionaries in the background (can be seen in the crash log after ^C).

I assume the reason is that the number of variants explodes without these standard filters.

Please notice that many of these filters usually don't have an impact on the tp-libvirt test functions but are rather tp-qemu specific. I took these filters to be the same as the ones the tp-libvirt/libvirt test type uses, where they possibly aren't functionally necessary either. I'm not sure.

In any case, as mentioned before, without these filters I never got the test to start in a reasonable time, so I propose let's keep them. :)